lib/cbra/cli.rb

```diff
@@ -14,5 +14,11 @@ def ls(app_path)
     def version
       puts Cbra::VERSION
     end
+
+    desc "graph APP_PATH", "Outputs graph.png to the current directory"
+    def graph(app_path)
+      Graph.new(app_path).png
+      puts "Graph generated in root directory"
```

Comment:
I'd expect to see `Graph generated at /Users/blangfeld/code/nitro-web/foo.png`. I'd like to be able to specify the filename, because `graph.png` isn't very descriptive.